lib/ansible/modules/cloud/docker/docker_swarm.py

```diff
@@ -416,15 +415,15 @@ def inspect_swarm(self):
             return
 
     def init_swarm(self):
-        if self.__isSwarmManager():
+        if not self.force and self.__isSwarmManager():
```

Comment:
This will overwrite #52886 changes, you need to rebase it

Reply:
I have to rebase it anyway since there are now conflicts ;)